app/views/lectures/show.html.erb

```diff
@@ -21,14 +21,16 @@
 <div class="tab-content" id="lectureTabsContent">
   <div class="tab-pane fade show active mx-auto" id="dashboard" role="tabpanel" aria-labelledby="dashboard-tab" style="max-width: 2000px;">
     <div class="row">
-      <div id="updates" class="col-12 col-sm-6 col-xl-4">
-          <% if @current_user.is_student %>
-            <h2 class="h5">Updates</h2>
-          <% else %>
-            <h2 class="h5">Questions</h2>
-          <% end %>
-          <%= react_component("UpdatesApp", {user_id: @current_user.id, is_student: @current_user.is_student, lecture_id: @lecture.id, course_id: @lecture.course.id, interactions_enabled: @lecture.allow_interactions?, questions_list: Question.questions_for_lecture(@lecture, current_user)}) %>
-      </div>
+      <% if @current_user.is_student %>
```

Comment:
Why do we only show updates when user is student? They should be also visible to lecturers if questions are enabled.